src/components/ebay-combobox/component.js

```diff
@@ -189,52 +193,61 @@ export default {
         this._cleanupMakeup();
     },
 
+    _setupFloatingLabel() { 
+        // TODO: makeup-floating-label should be updated so that we can remove the event listeners.
+        // It probably makes more sense to just move this functionality into Marko though.
+        if (this._floatingLabel) {
+            this._floatingLabel.refresh();
+            this.handleFloatingLabelInit();
+        } else if (document.readyState === 'complete') {
+            if (this.el) {
+                this._floatingLabel = new FloatingLabel(this.el);
+                this.handleFloatingLabelInit();
+            }
+        } else {
+            this.subscribeTo(window).once('load', () => {
+                this._setupFloatingLabel();
+            });
+        }
```

Comment:
I see a test failure on browser tests.
This whole thing needs to be wrapped in `if (this.input.floatingLabel) {` otherwise it runs each time even if there is no floating label. 
We can either wrap this whole thing in the if statement or the function call in `_setupMakeup`

Reply:
Yep. I got lost in the refactoring. I put back in. Glad the tests caught it.